### PR TITLE
Something something added something

### PR DESCRIPTION
**Pull request by Nicvaniek**

### CODE REVIEW

File: caesar.py [hunk 1]
```diff
@@ -4,9 +4,9 @@
             'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z',
             '0', '1', '2', '3', '4', '5', '6', '7', '8', '9']
```
Comment (AndreasNel) [inline caesar.py:5]: Why not put 0 after 9?

File: decrypt.py [hunk 2]
```diff
@@ -38,18 +39,33 @@
...
+# # Display results and if the answer is incorrect, show next 5
 print("Ciphertext: " + message)
 print("Best Guess Plaintext: " + best_effort)
 print("alpha key: " + str(len(alphabet) - exes[index]))
 print("digit key: " + str(len(alphabet) - jays[index]))
-# pprint.pprint(scores)
-finals.sort()
-pprint.pprint(finals)
-print(scores[best_effort])
+print("Number of options: " + str(scores[best_effort]))
```
Comment (AndreasNel) [inline decrypt.py]: I don't think this is correct....

**Review by AndreasNel — CHANGES_REQUESTED** [review]

File: caesar.py (hunk 1 above)
Comment (Nicvaniek) [reply caesar.py:5]: That was only neccessary if we were using one shift. Since we use a double shift it doesn't matter

File: decrypt.py (hunk 2 above)
Comment (Nicvaniek) [reply decrypt.py]: Apologies

**Pull request merged**